packages/react-testing/src/tests/element.test.tsx

```diff
@@ -458,9 +457,10 @@ describe('Element', () => {
   describe('#findAllWhere()', () => {
     it('finds all matching nodes', () => {
       const element = new Element(defaultTree, [divOne], defaultRoot);
-
       expect(
-        element.findAllWhere((element) => element === componentTwo),
+        element.findAllWhere(
+          (element) => ((element as unknown) as Element<{}>) === componentTwo,
+        ),
```

Comment:
Thanks to https://github.com/Shopify/quilt/pull/1999, I think you can do

```suggestion
        element.findAllWhere<Element<{}>>((element) => element === componentTwo),
```

Reply:
The generic type isn't applied to the predicate argument, only the return type. Should the implementation of `findWhere` and `findAllWhere` be changed?